mkfs/mkfs.c

```diff
@@ -609,6 +617,17 @@ int main(int argc, char *argv[])
 			}
 			ui.cluster_size = ret;
 			break;
+		case 'b':
+			ret = parse_size(optarg);
+			if (ret < 0)
+				goto out;
+			else if (ret & (ret - 1)) {
```

Comment:
Right. maybe, It is needed for cluster size, if yes, can you send another patch for this ?